hibernate-core/src/test/java/org/hibernate/test/where/hbm/LazyManyToManyNonUniqueIdNotFoundWhereTest.java

```diff
@@ -147,16 +150,16 @@ public void setup() {
 	public void cleanup() {
 		doInHibernate(
 				this::sessionFactory, session -> {
-					session.createSQLQuery( "delete from MATERIAL_RATINGS" ).executeUpdate();
-					session.createSQLQuery( "delete from BUILDING_RATINGS" ).executeUpdate();
-					session.createSQLQuery( "delete from ASSOCIATION_TABLE" ).executeUpdate();
-					session.createSQLQuery( "delete from MAIN_TABLE" ).executeUpdate();
+					session.createSQLQuery( getDialect().getDropTableString( "MATERIAL_RATINGS" ) ).executeUpdate();
+					session.createSQLQuery( getDialect().getDropTableString( "BUILDING_RATINGS" ) ).executeUpdate();
+					session.createSQLQuery( getDialect().getDropTableString( "ASSOCIATION_TABLE" ) ).executeUpdate();
+					session.createSQLQuery( getDialect().getDropTableString( "MAIN_TABLE" ) ).executeUpdate();
```

Comment:
This change is causing a failure with Oracle, it has to be reverted to
```
session.createSQLQuery( "delete from MATERIAL_RATINGS" ).executeUpdate();
session.createSQLQuery( "delete from BUILDING_RATINGS" ).executeUpdate();
session.createSQLQuery( "delete from ASSOCIATION_TABLE" ).executeUpdate();
session.createSQLQuery( "delete from MAIN_TABLE" ).executeUpdate();
```

Reply:
done